packages/react-overflow/CHANGELOG.json

```diff
@@ -1,33 +1,33 @@
 {
-  "name": "@fluentui/react-priority-overflow",
```

Comment:
tip: dunno how you did the rename, but in future you can use nx generator to automate things for you. this will also update all code references via AST, so you can be 100% sure you didn't miss anything 

https://nx.dev/packages/workspace/generators/move